x/incentives/module.go

```diff
@@ -148,31 +149,32 @@ func (am AppModule) RegisterServices(cfg module.Configurator) {
 	types.RegisterQueryServer(cfg.QueryServer(), keeper.NewQuerier(am.keeper))
 }
 
-// RegisterInvariants registers the capability module's invariants.
+// RegisterInvariants registers the module's invariants.
 func (am AppModule) RegisterInvariants(_ sdk.InvariantRegistry) {}
 
-// InitGenesis performs the capability module's genesis initialization It returns
-// no validator updates.
+// InitGenesis performs the module's genesis initialization.
+// It returns no validator updates.
+// TODO: What does "it returns no validator updates" mean?
```

Comment:
TODO: What does "it returns no validator updates" mean?

Reply:
Something dumb about EndBlock is that it returns validator updates. We should just delete that comment line

Reply:
Okay, instead of completely deleting these comments I will explain the way you have explained here and below. That way in case, for whatever reason if the sdk doesn't get to it, the next person who audits this wont be confused like I was. Thank you for explaining!